mcs/class/System/Mono.Net.Security/MonoTlsProviderFactory.cs

```diff
@@ -292,9 +292,9 @@ static void PopulateProviders ()
 #endif
 
 
-#if MONO_FEATURE_BTLS
+#if MONO_FEATURE_TLS
 		[MethodImpl (MethodImplOptions.InternalCall)]
-		internal extern static bool IsBtlsSupported ();
+		internal extern static bool IsTlsSupported ();
```

Comment:
This is one of the examples where this rename becomes more than just a cosmetic issue - since this would completely change the meaning of what the function does, the new name is unacceptable in this context.